### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by tshqin**
PTAL @jomenxiao

### CODE REVIEW

File: metrics/snapshot-collector/README.md [hunk 1]
```diff
@@ -2,12 +2,21 @@ metrics-snapshot-collector
...
+- **Make sure grafana server has shared object file `libfontconfig.so.1`**
 - debian/ubuntu:  
 	- `apt-get install -y libfontconfig freetype-devel fontconfig-devel fontconfig`
 - centos: 
 	- `yum install -y fontconfig freetype freetype-devel fontconfig-devel libstdc++`
+- **Make sure grafana server has fonts installed for English**
+- to list the fonts support English: 
+	- `fc-list :lang=en`
+- if the output is empty, you will need to install at least one font that supports English, here is an example of install Google Fonts on Linux servers: 
+	- `cd`
```
Comment (jomenxiao) [inline metrics/snapshot-collector/README.md]: change directory to?

File: metrics/snapshot-collector/README.md [hunk 2]
```diff
@@ -2,12 +2,21 @@ metrics-snapshot-collector
...
 	- `yum install -y fontconfig freetype freetype-devel fontconfig-devel libstdc++`
+- **Make sure grafana server has fonts installed for English**
+- to list the fonts support English: 
+	- `fc-list :lang=en`
+- if the output is empty, you will need to install at least one font that supports English, here is an example of install Google Fonts on Linux servers: 
+	- `cd` 
+	- `sudo wget https://github.com/google/fonts/archive/master.zip` 
+	- `unzip master.zip` 
+	- `sudo cp -rvf fonts-master /usr/share/fonts` 
+	- `fc-cache -fv`
```
Comment (jomenxiao) [inline metrics/snapshot-collector/README.md]: sudo

File: metrics/snapshot-collector/README.md (hunk 1 above)
Comment (tshqin) [reply metrics/snapshot-collector/README.md]: $HOME - just for instance

File: metrics/snapshot-collector/README.md (hunk 2 above)
Comment (tshqin) [reply metrics/snapshot-collector/README.md]: I tested, doesn't need sudo for fc-cache -fv

**Comment by tshqin** [comment]
PTAL @iamxy

**Comment by iamxy** [comment]
LGTM

**Pull request merged**